### PR TITLE
Cadence HiFi 5 NN Library v1.5.0 integration

### PR DESCRIPTION
**Pull request by bhanuprakashbv**
Integrate the HiFi 5 Neural Network Library v1.5.0
Used HiFi 5 NN Library kernels for conv, depthwise_conv and softmax
operators int8 variant.

Signed-off-by: Anirban Mandal <amandal@cadence.com>
Signed-off-by: Biswa Dhal <bdhal@cadence.com>
Signed-off-by: Bhanu Prakash Bandaru Venkata <bhanup@cadence.com>
Signed-off-by: Chaitanya Sanjay Muley <cmuley@cadence.com>
Signed-off-by: Harinarayanan E V <hariev@cadence.com>
Signed-off-by: Harshavardhan Ravindra Joshi <joshih@cadence.com>
Signed-off-by: Prasad Nikam <pnikam@cadence.com>

### CODE REVIEW

**Comment by nyadla-sys** [comment]
@advaitjain :Please merge this branch

**Review by advaitjain — APPROVED** [review]

**Comment by advaitjain** [comment]
> @advaitjain :Please merge this branch

I have pushed a commit with some updated formatting and small name changes. This PR is now ready to be merged.
